pkgs/development/python-modules/humanize/default.nix

```diff
@@ -12,7 +12,7 @@
 
 buildPythonPackage rec {
   pname = "humanize";
-  version = "4.6.0";
+  version = "4.7.0";
   format = "pyproject";
 
   disabled = pythonOlder "3.7";
```

Comment:
```suggestion
  disabled = pythonOlder "3.8";
```

Reply:
The conditional dependency on importlib-metadata can also be dropped, otherwise LGTM

Reply:
Thanks for noticing!